backend/lucene/src/main/java/org/hibernate/search/backend/lucene/logging/impl/Log.java

```diff
@@ -705,4 +705,13 @@ SearchException cannotHighlightInNestedContext(String currentNestingField,
 	@Message(id = ID_OFFSET + 173, value = "'%1$s' cannot be nested in an object projection. "
 			+ "%2$s")
 	SearchException cannotUseProjectionInNestedContext(String projection, String hint, @Param EventContext eventContext);
+
+	@Message(id = ID_OFFSET + 174,
+			value = "Vector '%1$s' cannot be equal to '%2$s'. It must be a positive integer value not greater than %3$s.")
+	SearchException vectorPropertyUnsupportedValue(String property, Integer value, int max);
+
+	@LogMessage(level = Level.WARN)
+	@Message(id = ID_OFFSET + 175,
+			value = "Vector field configuration property '%1$s' value '%2$s' is out of the recommended range [%3$d, %4$d]")
+	void vectorPropertyOutOfRecommendedRange(String property, Integer value, int min, int max);
```

Comment:
Not sure such warning makes sense.

Either you think values out of the range will work fine and you don't warn, or you think these values will cause problems and then you just throw an exception.

If values out of the range may work, but have downsides, you should just make the accepted range configurable (through backend properties). Though this is definitely out of scope for this PR, so would rather be a follow-up Jira issue.

Reply:
yeah ... I've seen that there are these recommendations on the ranges for these values, so I thought it would be nice to show it to the user. And it is targeting the users who aren't sure about what to pick and are just "playing around" 😄. From what I understand, things might still work fine if the values are out of the range.... but maybe it would be ok to just move it to the docs, or Javadocs and remove the log message.